be/src/service/http_service.cpp

```diff
@@ -176,6 +177,13 @@ Status HttpService::start() {
     _ev_http_server->register_handler(HttpMethod::POST, "/api/list_resource_groups", list_workgroup_action);
     _http_handlers.emplace_back(list_workgroup_action);
 
+    RuntimeFilterCacheAction* runtime_filter_cache_action = new RuntimeFilterCacheAction(_env);
```

Comment:
Who need to observe this metric? DBA ? User? Developer?

Reply:
Developer